### ISSUE TITLE
Use `AppSettings::RequireHelp`

### ISSUE THREAD

**Issue by casey** [casey/intermodal#158]
This [was implemented](https://github.com/clap-rs/clap/issues/1639) in clap master, but to use it, it'll have to make its way into structopt.

**Comment by pksunkara** [comment]
structopt won't be adopting clap master. It's only for clap v2. Clap master is clap v3. And it has clap_derive which was basically structopt.

**Comment by casey** [comment]
Thanks for the info! I assumed that structopt might eventually migrate to clap V3 once it was released, is that not the case?

**Comment by pksunkara** [comment]
Nope. Documentation will be updated about this info eventually, but structopt is limited to only clap v2. clap_derive is for clap v3. We basically combined the efforts. You can think of clap v3 as (clap v3 + structopt v3)

**Comment by casey** [comment]
Thanks for the color. Awesome, looking forward to clap v3!
